how-to/workspace-platform-starter/public/manifest.fin.json

```diff
@@ -1174,7 +1174,7 @@
 					"enabled": false,
 					"url": "http://localhost:8080/js/modules/actions/favorites-menu.bundle.js",
 					"data": {
-						"menuType": "native"
+						"menuStyle": "custom"
```

Comment:
we have to keep native as default for now until we discuss it

Reply:
That one slipped through from my testing, fixed